surveys/templates/registration/login.html

```diff
@@ -19,22 +22,14 @@
 
     <form method="post" action="{% url 'login' %}">
     {% csrf_token %}
-    <table>
-    <tr>
-        <td>{{ form.username.label_tag }}</td>
-        <td>{{ form.username }}</td>
-    </tr>
-    <tr>
-        <td>{{ form.password.label_tag }}</td>
-        <td>{{ form.password }}</td>
-    </tr>
-    </table>
 
-    <input type="submit" value="login" class="btn btn-success float-right">
+    {{form.username | as_crispy_field}}
+    {{form.password | as_crispy_field}}
```

Comment:
These inputs are a bit too wide for my taste on large screens:

![Screenshot from 2019-05-22 10-18-09](https://user-images.githubusercontent.com/14170650/58182121-08aeb600-7c7b-11e9-9a09-939b3831af61.png)

Reply:
Good point!